integration-tests/src/main/java/org/infinispan/quarkus/embedded/TestServlet.java

```diff
@@ -143,4 +147,30 @@ public String create(@PathParam("id") String id, Book book) {
         books.put(id, book);
         return id;
     }
+
+    @Path("ANNOTATIONS/GET/review/{bookId}")
+    @GET
+    @Produces(MediaType.TEXT_PLAIN)
+    public String create(@PathParam("bookId") String bookId) {
+        Objects.requireNonNull(bookId);
+        return reviewService.getReview(bookId);
+    }
+
+    @Path("ANNOTATIONS/GET/clear/{bookId}")
+    @GET
```

Comment:
Is this really a GET operation? Shouldn't it be a DELETE or POST?

Reply:
this is all in get manners for the testing purpose only